ang/crmMailing/BlockMailing.html

```diff
@@ -79,5 +79,19 @@
         placeholder="Subject"
         name="subject" />
     </div>
+    <div ng-if="crmMailingConst.isMultiLingual">
+      <div crm-ui-field="{name: 'subform.language', title: ts('Language')}">
+        <select
+          crm-ui-id="subform.language"
+          crm-ui-select="{dropdownAutoWidth : true, allowClear: false}"
+          name="language"
+          ng-model="mailing.language"
+          required
+          >
+          <option value=""></option>
```

Comment:
Yes, `- choose language -` is the best thing to do. We want people to be aware of the choice they make. In the future, we might want to define default value for this but i don't think it's required for now.